vale/code/crypto/aes/Vale.AES.Types_helpers_BE.fsti

```diff
@@ -0,0 +1,126 @@
+module Vale.AES.Types_helpers_BE
+
+open Vale.Def.Opaque_s
+open Vale.Def.Words_s
+open Vale.Def.Words.Seq_s
+open Vale.Def.Words.Four_s
+open Vale.Def.Types_s
+open Vale.Arch.Types
+open FStar.Mul
+open FStar.Seq
+open FStar.Math.Lemmas
+open Vale.Lib.Seqs
+open Vale.Lib.Seqs_s
+open Vale.Def.Words.Seq
+
+unfold let pow2_24 = 0x1000000
+let nat24 = natN pow2_24
+
+val seq_four_to_seq_to_seq_four_BE (#a:Type) (x:seq a{length x % 4 == 0}) :
```

Comment:
It seems that there are similar lemmas already in, e.g., Vale.Arch.Types and Vale.Def.Words.Seq. Could we consolidate all lemmas on sequences and vale types there to avoid code duplication in the future?

Reply:
I agree. Lemmas consolidated now to Vale.Arch.Types and Vale.Def.Words.Seq modules.